eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -31,6 +31,15 @@ steps:
     displayName: Build
 
   - ${{ if ne(parameters.skipTests, 'true') }}:
+    - ${{ if ne(parameters.isWindows, 'true') }}:
+      - script: mkdir ${{ parameters.repoArtifactsPath }}/devcert-scripts &&
+                cd ${{ parameters.repoArtifactsPath }}/devcert-scripts &&
+                wget https://raw.githubusercontent.com/BorisWilhelms/create-dotnet-devcert/main/scripts/ubuntu-create-dotnet-devcert.sh &&
+                wget https://raw.githubusercontent.com/BorisWilhelms/create-dotnet-devcert/main/scripts/common.sh &&
+                chmod +x ubuntu-create-dotnet-devcert.sh &&
+                ./ubuntu-create-dotnet-devcert.sh
```

Comment:
This seems risky as we can get broken (or compromised) by updates to this repo that we don't own.

Reply:
I opened https://github.com/dotnet/dnceng/issues/3123 to move this to the image itself, so it gets done only once. We have been using this for the helix tests already.